README.md

```diff
@@ -437,6 +437,7 @@ Massachusetts Bay Transportation Authority (MBTA)](https://github.com/mbta).
 - [General Bikeshare Feed Specification (GBFS)](https://github.com/NABSA/gbfs) - Open data standard for real-time bikeshare information developed by members of the [North American Bikeshare Association (NABSA)](http://nabsa.net/).
     - [gbfs-validator](https://github.com/PierrickP/gbfs-validator) - 3rd party tool to validate GBFS feeds.
     - [gbfs R package](https://github.com/ds-civic-data/gbfs) - Functions to interface with GBFS feeds in R, allowing users to save and accumulate tidy .rds datasets for specified cities/bikeshare programs.
+    - [GBFS Documentation Platform](https://gbfs.mobilitydata.org/) - Documentation platform for the General Bikeshare Feed Specification. View the GBFS specification, change proposals, policy guides, and technical tools.
```

Comment:
Thanks for this! Could you add a note that this is maintained by MobilityData and whether or not this is "official" (i.e., endorsed/supported by NABSA)?